Sources/TokenizersWrapper/TokenizersWrapper.swift

```diff
@@ -0,0 +1,83 @@
+import Foundation
+import Hub
+
+@_exported import TokenizersCore
+
+#if canImport(TokenizersTemplates)
+import TokenizersTemplates
+public typealias PreTrainedTokenizer = PreTrainedTokenizerWithTemplates
+#endif
```

Comment:
So if `TokenizersTemplates` is available (because users have declared it as a dependency), we override the definition so the factory below uses the subclass.